examples/example_pump_data_model/transformations/pump_model-populate-lift_station_pumps_edges.sql

```diff
@@ -0,0 +1,8 @@
+select
+  concat(cast(`parent`.externalId as STRING), ':', cast(`externalId` as STRING)) as externalId,
+  `parent` as startNode,
+  node_reference('{{instance_space}}', cast(`externalId` as STRING)) as endNode
+from
+  cdf_data_models("{{source_model_space}}", "{{source_model}}", "1", "Asset")
+where
+  startswith(title, 'Pump')
```

Comment:
Should this be a variable? Either Pump or maybe even the full where statement for how to pick out pumps?

Reply:
this is a bit particular part of the data set. I would ideally use the external id, but there was a bug in transformations preventing me from doing that (reported to them)